### PR TITLE
leader-board setup

### PR DESCRIPTION
**Pull request by husush23**
### In this project I:
built the setup of the leader-board project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,107 @@
...
+
+I would like to thank everyone who encourages me doing all these tasks.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (Zuhaib042) [inline README.md:103]: - [ ] This link is broken, kindly consider fixing this link by adding `MIT.md` in your main directory and consider checking if its working before submitting again. having broken links in our repo make it unprofessional.

File: README.md [hunk 2]
```diff
@@ -1 +1,107 @@
...
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Leader board <a name="about-project"></a>
+
+**My portfolio** is a project based on first mobile structure and it is complete portfolio with even desktop responsive version.
```
Comment (Zuhaib042) [inline README.md]: - [ ] This description of the project is not relevant to this project, kindly consider updating it with relevant information.

File: README.md [hunk 3]
```diff
@@ -1 +1,107 @@
...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>portfolio website</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
```
Comment (Zuhaib042) [inline README.md]: - [ ] This description of the project is not relevant to this project, kindly consider updating it with relevant information like your stack should be `HTML, CSS, JavaScript, webpack`

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @husush23 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Perfectly set up the project with webpack
✅ Correct Gitflow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @husush23  👋 

### [OPTIONAL]
In the Future, Please write descriptive commit messages and use Imperative sentences for writing commit messages because it's considered as Best Practice in the tech industry. 👍 

Screenshot 2023-05-22 at 9 19 27 PM

Screenshot 2023-05-22 at 9 19 51 PM


## Things to Appreciate 
- [ ] Correct Flow ✅ 
- [ ] Clean Code 👍 
- [ ] No Linter Errors 🔥 
- [ ] Descriptive Pull Request and Summary 👌 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**